maintenance/scripts/refresh_auth.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+#setup nvm
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
+cd /var/www/html/auth
+composer install
+php artisan migrate
+php artisan config:cache
+nvm install --latest-npm
```

Comment:
The [docs are not super clear](https://github.com/nvm-sh/nvm#migrating-global-packages-while-installing), but `nvm install --latest-npm` is meant to install the version of node specified (or if not, in the .nvmrc file) and update npm at the same time.

If we don't want a one liner, then it would be:
```
nvm use
nvm install-latest-npm
```

Reply:
Yep, I think @mnigh is correct.

Reply:
Ah, then that's fine